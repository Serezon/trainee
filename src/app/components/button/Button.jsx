import React from 'react'

const Button = ({ logout }) => <button type="button" onClick={() => logout()}>Logout</button>

export default Button
