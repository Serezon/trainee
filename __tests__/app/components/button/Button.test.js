/* eslint-disable */
import React from 'react'
import renderer from 'react-test-renderer'

import Button from '../../../../src/app/components/button/Button'

describe('>>> Button --- Snapshot', () => {
  it('+++capturing snapshot of Button', () => {
    const renderedValue = renderer.create(<Button logout={[Function]}/>).toJSON()
    expect(renderedValue).toMatchSnapshot()
  })
})

describe('Testing Storybook', () =>{
  it('simple test', () => {
    expect(2+2).toBe(4)
  })
})
