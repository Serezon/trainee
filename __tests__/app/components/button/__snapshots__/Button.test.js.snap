// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Button --- Snapshot +++capturing snapshot of Button 1`] = `
<button
  onClick={[Function]}
  type="button"
>
  Logout
</button>
`;
